handel/src/services/iot/params-schema.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "The service type (should always be 'iot')",
+            "pattern": "iot",
+            "errorMessage": "The 'type' field must equal a 'iot'"
+        },
+        "description": {
+            "type": "string",
+            "errorMessage": "The 'description' field be a string"
+        },
+        "event_consumers": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "service_name": {
+                        "type": "string",
+                        "errorMessage": "The 'service_name' parameter is required in each config in the 'event_consumers' section"
+                    },
+                    "sql": {
+                        "type": "string",
+                        "errorMessage": "The 'sql' parameter is required in each config in the 'event_consumers' section"
+                    }
+                },
+                "required": [
+                    "service_name",
+                    "sql"
+                ],
+                "errorMessage": {
+                    "required": {
```

Comment:
You'll want to add an error message for "additionalProperties" here too in addition to the one you already have for "required".